projects/gloo/cli/pkg/cmd/initpluginmanager/initpluginmanager.go

```diff
@@ -51,7 +51,7 @@ Add the glooctl plugins to your path with:
   export PATH=%s/bin:$PATH
 Now run:
   glooctl plugin --help     # see the commands available to you
-Please see visit the Gloo Edge website for more info:  https://www.solo.io/products/gloo-edge/
+Please see visit the Gloo Gateway website for more info:  https://www.solo.io/products/gloo-edge/
```

Comment:
maybe update to 
```suggestion
Please see visit the Gloo Gateway website for more info: https://docs.solo.io/gloo-edge/
```

I think i missed this on your pr on main btw you might want to add this there too